### ISSUE TITLE
Ledvance Plug Compact EU EM T, 4099854293276, show as Unsupported

### ISSUE THREAD

**Issue by Johlar76** [Koenkk/zigbee2mqtt#24362]
### What happened?

The plug show as Unsupported after joined.

### What did you expect to happen?

As far as I can understand from the Supported Devices list, it should be supported.

### How to reproduce it (minimal and precise)

Disable join in Z2M
Delete device from Z2M
Reset device (hold button for >10 sec until blinking)
Unplug device
Permit join in Z2M
Connect device

### Zigbee2MQTT version

1.40.2

### Adapter firmware version

20220219

### Adapter

SONOFF Zigbee 3.0 USB Dongle Plus ZBDongle-P

### Setup

RPI4, Raspbian GNU/Linux 11 (bullseye), Z2M installed on Linux

### Debug log

[log_2.log](https://github.com/user-attachments/files/17401889/log_2.log)

**Comment by Koenkk** [comment]
Will be supported in the next z2m release!

**Comment by Johlar76** [comment]
Ok, thanks!

**Comment by grobihh** [comment]
Great. I was about to file an issue.
The device seems to works without problems. Only it is reported as UNSUPPORTED and the picture is missing in 1.40.2.

**Comment by MrBlue90** [comment]
Confirmed that the new version of Z2M officially supports the plug. Thanks @Koenkk !

**Comment by Johlar76** [comment]
Yes. Works perfect with 1.41.0. Thanks @Koenkk !

**Comment by dimafemabo** [comment]
I tried to integrate the device, which I got today April 5th and I got the same error.
Not supported device 
Zigbee2mqtt ver. 1.42.0-2
Device doesn't show in the dashboard.

Log im HA:
[2025-04-05 14:14:31] error: 	zh:controller: Interview failed for '0xf0d1b8be240b4081 with error 'Error: Interview failed because can not get node descriptor ('0xf0d1b8be240b4081')'
[2025-04-05 14:14:31] error: 	z2m: Failed to interview '0xf0d1b8be240b4081', device has not successfully been paired
[2025-04-05 14:14:31] info: 	z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xf0d1b8be240b4081","ieee_address":"0xf0d1b8be240b4081","status":"failed"},"type":"device_interview"}'
[2025-04-05 14:14:33] info: 	zh:controller: Interview for '0xf0d1b8be240b4081' started
[2025-04-05 14:14:33] info: 	z2m: Starting interview of '0xf0d1b8be240b4081'
[2025-04-05 14:14:33] info: 	z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xf0d1b8be240b4081","ieee_address":"0xf0d1b8be240b4081","status":"started"},"type":"device_interview"}'

Any ideas?
Many thanks for pointing me into the right direction.
